### ISSUE TITLE
Unable to disable protection using doorhangar

### ISSUE THREAD

**Issue by pdehaan** [biancadanforth/tracking-protection-shield-study#69]
### Steps to reproduce:

1. Opt into the "fast" variation (probably not relevant).
1. Navigate to http://www.msnbc.com/
1. Click on the shield icon in the address bar (it should say something like 0 trackers blocked and 0 seconds saved). (see Figure 1 below)
1. Click "Disable Protection" button. (See figure 1 below)
1. Click the "Disable" button. (See figure 2 below)
1. Click the shield icon in the address bar (it should say something like "Tracking Protection: Enabled" still). (See figure 1 below)

---

![msnbc1](https://user-images.githubusercontent.com/557895/35541753-23a5187e-0511-11e8-83fe-4cb706de4310.png)
**Figure 1**

---

![msnbc2](https://user-images.githubusercontent.com/557895/35541783-3eaf6a8e-0511-11e8-913d-c0b32f766dad.png)
**Figure 2**

**Comment by biancadanforth** [comment]
@pdehaan I cannot reproduce this...

I noticed you have built-in tracking protection turned on (the shield icon on the left-hand side of the URL bar). I recently merged a [PR](https://github.com/biancadanforth/tracking-protection-shield-study/pull/66) to implement eligibility requirements for the study. This should prevent you from participating in the study in this case. Did you fetch the latest source code when you ran this?

When I try these steps (with default TP settings), the icon goes away, the addon uninstalls and built-in TP is reset to its default setting in `about:preferences`. When I try these steps with default TP enabled globally, the study ends before any UI is shown.
